sharktank/sharktank/ops/custom_impls.py

```diff
@@ -146,6 +147,40 @@ def matmul_generic_tensor_block_scaled_fp4(
     )
 
 
+@matmul.override(Tensor, QuantizedTensor)
```

Comment:
This is generally correct, but because of how the dispatch system works, the behavior might not be desirable; for now could you:
- Comment this line
- Add: `TODO(kyleherndon): Re-enable this after creating a matmul kernel selector`